VxWorks/Os/Console.hpp

```diff
@@ -14,8 +14,8 @@ namespace Console {
 //! ConsoleHandle class definition for VxWorks implementations.
 //!
 struct VxWorksConsoleHandle : public ConsoleHandle {
-    //! VxWorks console file descriptor
-    FILE* m_file_descriptor = stdout;
+    char circularBuffer[MAX_CONSOLE_CAPACITY][MAX_CONSOLE_MESSAGE_BYTE_SIZE];  // Circular buffer to store messages
+    FwIndexType m_tail_index = 0;  // Index pointing to the tail of the circular buffer
```

Comment:
This should be an atomic variable, otherwise two events could compete for the circular buffer slot.